javascript/statements.json

```diff
@@ -1347,12 +1347,8 @@
                 "version_added": false
               },
               "oculus": "mirror",
-              "opera": {
-                "version_added": false
-              },
-              "opera_android": {
-                "version_added": false
-              },
+              "opera": "mirror",
+              "opera_android": "mirror",
```

Comment:
I have not tested this, but I believe it to be true. 
1. Opera follows chrome in this release (normally). So would need specific reason not to support this feature.
2. This is identical "feature" to _Support for ECMAScript modules_ in https://developer.mozilla.org/en-US/docs/Web/API/Worker/Worker#browser_compatibility - where Opera does follow Chrome.

So definitely one of the two entries is wrong, and it makes more sense that it is this one.

Reply:
This was probably a case of Opera not getting updated when Chrome added support due to a delayed release, I don't see any reason Opera would explicitly disable support.

Reply:
Me either!